src/index.js

```diff
@@ -1,3 +1,8 @@
+import 'core-js/modules/es7.array.includes';
```

Comment:
Try moving these after `import { HashRouter as Router } from 'react-router-dom';`

Reply:
I missed "@import 'carbon-components/scss/globals/scss/vendor/@carbon/type/scss/font-family.scss';" . It works now